get-started/index.md

```diff
@@ -9,15 +9,33 @@ layout: posts
 Cryostat {{ site.data.versions.cryostat.version }}
 
 ## [Installing Cryostat Operator](#installing-cryostat-operator)
-Coming soon to [OperatorHub](https://operatorhub.io/). In the meantime, you can install
-the Cryostat Operator using kubectl, or by deploying the bundle image with Operator SDK.
-
+To install the Cryostat Operator, you can choose one of the following installation options: install via OperatorHub (recommended), install with kubectl, or install by deploying the bundle image with Operator SDK. All of the installation methods require cert-manager as a prerequisite.
```

Comment:
Maybe it's just too much complexity to add to the Getting Started guide, but should we document here that it is actually possible (though not recommended unless you have good reason) to deploy without cert-manager and disable the integration for it?

Reply:
If you think some new users would need to deploy without cert-manager, maybe I can add a note with the link to https://github.com/cryostatio/cryostat-operator/blob/v2/docs/config.md#disabling-cert-manager-integration at the bottom of the "Install cert-manager" section. What do you think?

Reply:
That's a good idea but I think we should still have some stronger disclaimer that we don't recommend that - it should only really be done because cert-manager is not available *and* there is some other solution for encrypted traffic instead, or the user 100% trusts everything that runs in the same cluster (ie it's fully under their control, not a shared or public cloud instance).